### PR TITLE
Improve indexify examples

### PR DESCRIPTION
**Pull request by PulkitMishra**
# Improve indexify examples

### CODE REVIEW

**Comment by PulkitMishra** [comment]
what do we want to do with indexify/examples/website_audio_summary ,  indexify/examples/tweetsgenerator, indexify/examples/multi_model_pdf_parser and indexify/examples/image_detection_indexing ?

File: examples/video_summarization/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+2. Install the required dependencies:
+   ```
+   pip install -r requirements.txt
+   ```
+
+## Usage
+
+1. Set up environment variables:
+   ```
+   export INDEXIFY_API_KEY=your_indexify_api_key
```
Comment (diptanu) [inline examples/video_summarization/README.md:37]: We don't have this yet

File: examples/video_summarization/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+   ```
+
+2. Run the main script:
+   ```
+   python youtube_summarizer.py
```
Comment (diptanu) [inline examples/video_summarization/README.md:42]: Did you test if this works? I think by default it tries to run this with the server?

**Review by diptanu — CHANGES_REQUESTED** [review]

**Comment by PulkitMishra** [comment]
@diptanu except for website audio (didnt have elevenlabs credits) rest are tested

File: examples/knowledge_graph/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+FROM python:3.9-slim
```
Comment (diptanu) [inline examples/knowledge_graph/Dockerfile:1]: Delete this file as discussed offline.

File: examples/knowledge_graph/README.md [hunk 4]
```diff
@@ -13,42 +13,66 @@ This project demonstrates how to build a Knowledge Graph Retrieval-Augmented Gen
...
    ```
 
-## Usage
+6. Run the main script:
+   ```
+   python kg_rag_qa_pipeline.py
+   ```
 
-1. Ensure your Neo4j database is running and accessible.
+### Option 2: Using Docker Compose
```
Comment (diptanu) [inline examples/knowledge_graph/README.md:55]: There is an additional step to build the images -

```
indexify-cli build-image workflow.py NLPFunction
indexify-cli build-image workflow.py generate_embeddings
indexify-cli build-image workflow.py  store_in_neo4j
indexify-cli build-image workflow.py  generate_embeddings
```

File: examples/knowledge_graph/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
+version: '3.8'
+
+services:
+  app:
```
Comment (diptanu) [inline examples/knowledge_graph/docker-compose.yml:4]: This needs the container for the 1. server 2. All the images that are required to run the functions.

File: examples/video_summarization/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
```
Comment (diptanu) [inline examples/video_summarization/docker-compose.yml:7]: See the comment above about dockerfile

File: examples/website_audio_summary/docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
```
Comment (diptanu) [inline examples/website_audio_summary/docker-compose.yml:7]: Same as above

**Review by diptanu — CHANGES_REQUESTED** [review]

**Pull request merged**